modules/gumgumBidAdapter.js

```diff
@@ -198,6 +199,9 @@ function buildRequests (validBidRequests, bidderRequest) {
     if (data.gdprApplies) {
       data.gdprConsent = gdprConsent.consentString;
     }
+    if (uspConsent) {
+      data.uspConsent = uspConsent;
```

Comment:
the param should be `us_privacy`

Reply:
actually, although the IAB spec for USP/CCPA suggests `us_privacy` as the industry standard, you're not required to use that parameter for your endpoint.